test/e2e/app-dir/server-source-maps/server-source-maps.test.ts

```diff
@@ -100,13 +100,12 @@ describe('app-dir - server source maps', () => {
           ? '\nError: Boom' +
               '\n    at logError (app/ssr-error-log-ignore-listed/page.js:9:16)' +
               '\n    at runWithInternalIgnored (app/ssr-error-log-ignore-listed/page.js:19:12)' +
-              // TODO(veil): Turbopack's sourcemap loader drops `ignoreList`
-              '\n    at runInternalIgnored (internal-pkg/ignored.ts:6:9)' +
               '\n    at runWithExternalSourceMapped (app/ssr-error-log-ignore-listed/page.js:18:29)' +
               '\n    at runWithExternal (app/ssr-error-log-ignore-listed/page.js:17:32)' +
               '\n    at runWithInternalSourceMapped (app/ssr-error-log-ignore-listed/page.js:16:18)' +
               // Realpath does not point into node_modules so we don't ignore it.
-              '\n    at runInternalSourceMapped (internal-pkg/sourcemapped.ts:5:9)' +
+              // TODO(veil): Should be internal-pkg/sourcemapped.ts
+              '\n    at runInternalSourceMapped (sourcemapped.ts:5:9)' +
```

Comment:
This is a regression. The sourcemap contains `sourcemapped.ts` in `sources`. This should be handled as a relative URL.